src/api/config/certs.yml

```diff
@@ -0,0 +1,11 @@
+tls:
+  certificates:
+    - certFile: /certs/path/to/certs
```

Comment:
Why does one line have a leading `-` and the next one not?

Also, are these paths real or meant to be place holders?  If the latter, can you leave comments explaining?

Reply:
> Why does one line have a leading - and the next one not?
I was following the same format they use in [their documentation](https://doc.traefik.io/traefik/v2.0/https/tls/#user-defined).

> Also, are these paths real or meant to be place holders? If the latter, can you leave comments explaining?
:+1:  I'll leave a coment

Reply:
Can we do this in the docker-compose `production.yml` with labels instead of here in a separate file, so it can use the env vars in scope there?

Reply:
@humphd That hasn't been implemented yet (but there's a lot of demand for it)

https://github.com/traefik/traefik/issues/5507

Reply:
Interesting, I didn't realize you had to have an external file for this.  Seems odd.  Thanks for the link.